### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by veggiemonk**
* One single file for the build
* Remove single link reference
* Fix lower case / upper case mismatch
* Add GitHub emoji to description
* Add category in the table

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -94,17 +117,24 @@ const formatEntry = (
     description,
     homepage,
     stargazers_count: stargazers,
+    // subscribers_count: watchers,
```
Comment (agebhar1) [inline build.js]: consider to remove code

File: build.js [hunk 2]
```diff
@@ -62,6 +66,25 @@ const sitemapOpts = {
...
+
+let emojiMapURL = {};
+
+const emojify = text => {
+  if (!text) return text;
+  const colonWrapped = /(:[\w\-+]+:)/g;
+  const result = text.replace(colonWrapped, match => {
+    const name = match.replace(/:/g, '');
+    const url = emojiMapURL[name];
+    return `<img src="${url}" class="emoji" alt="${name}" />`;
```
Comment (agebhar1) [inline build.js]: `url` may be `null` if map doesn't contain key/emoji

File: build.js [hunk 3]
```diff
@@ -224,6 +292,7 @@ const bundle = () => {
 };
 
 async function main() {
+  emojiMapURL = await loadEmoji();
```
Comment (agebhar1) [inline build.js]: consider to remove global var `emojiMapURL` by injecting into processing function(s)

Comment (veggiemonk) [reply build.js]: Well, it's true that global variable are considered a bad practice. But this is to be considered as a function that just happened to be a HashMap. That's why I used it that way.

File: build.js (hunk 2 above)
Comment (veggiemonk) [reply build.js]: good catch 👍

File: build.js (hunk 1 above)
Comment (veggiemonk) [reply build.js]: You're totally right 👍

File: build.js (hunk 3 above)
Comment (agebhar1) [reply build.js]: That's fine.

**Pull request merged**